bigtop_toolchain/manifests/packages.pp

```diff
@@ -293,21 +293,10 @@
         "python-setuptools",
         "libffi-dev",
         "python3-dev",
-        "python2.7-dev",
         "nasm",
         "yasm"
       ]
-      if (($operatingsystem == 'Ubuntu' and 0 <= versioncmp($operatingsystemmajrelease, '22.04'))) {
-        file { '/usr/bin/python':
-          ensure => 'link',
-          target => '/usr/bin/python2',
-        }
-        $pkgs = $_pkgs
-      } elsif (($operatingsystem == 'Ubuntu' and 0 <= versioncmp($operatingsystemmajrelease, '20.04')) or ($operatingsystem == 'Debian' and 0 <= versioncmp($operatingsystemmajrelease, '11'))) {
-        $pkgs = concat($_pkgs, ["python-is-python2"])
-      } else {
-        $pkgs = $_pkgs
-      }
+      $pkgs = $_pkgs
```

Comment:
We should remove no longer needed temporary `_pkgs` now. @JiaLiangC.

Reply:
@iwasakims  already remove redundant variable $_pkgs